### PR TITLE
Automate Edit Sequence of user groups

### PR DESCRIPTION
**Pull request by akarol**
{{pytest: -v -k 'test_edit_sequence_usergroups'}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 5a5f273ea492dbd56ec0e270a3d1e3634319501c:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d15c03348a7b9c9933c42b3243c80e044011f00b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/configure/test_access_control.py [hunk 1]
```diff
@@ -286,6 +287,19 @@ def test_edit_default_group():
     flash.assert_message_match(flash_msg.format(group.description))
 
 
+def test_edit_sequence_usergroups():
```
Comment (mfalesni) [inline cfme/tests/configure/test_access_control.py]: Maybe some docstring here?

File: cfme/tests/configure/test_access_control.py [hunk 2]
```diff
@@ -286,6 +287,19 @@ def test_edit_default_group():
...
+    group = new_group()
+    group.create()
+    sel.force_navigate("cfg_accesscontrol_groups")
+    row = group_table.find_row_by_cells({'Name': group.description})
+    original_sequence = sel.text(row.sequence)
+    set_group_order([group.description])
+    row = group_table.find_row_by_cells({'Name': group.description})
+    changed_sequence = sel.text(row.sequence)
+    assert original_sequence != changed_sequence, "Edit Sequence Failed"
+    group.delete()
```
Comment (mfalesni) [inline cfme/tests/configure/test_access_control.py]: `request.addfinalizer(group.delete)` at the beginning.

Comment (mfalesni) [inline cfme/tests/configure/test_access_control.py]: After the create of course :)

**Comment by dajoRH** [comment]
Lint report for commit ebe61e080371a08ee77dd253ca45b309351f3bdc:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/configure/test_access_control.py (hunk 1 above)
Comment (akarol) [inline cfme/tests/configure/test_access_control.py]: @mfalesni  Added!

**Comment by mfalesni** [comment]
The upstream ran last run which was no different from the functional side, seems like it failed on appliance request. Merging.

**Pull request merged**